### PR TITLE
Support icon in avatar component

### PR DESCRIPTION
**Pull request by remy727**
### WHY are these changes introduced?

Improve `Avatar` component

### WHAT is this pull request doing?

Support icon in `Avatar` component

### Usage

```html.erb
<%= polaris_avatar(icon: "StoreIcon") %>
```

### CODE REVIEW

**Review by kirillplatonov — CHANGES_REQUESTED** [review]
All good. Let's just add lookbook preview for this new feature.

File: lib/polaris/view_components/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Polaris
   module ViewComponents
-    VERSION = "2.1.0"
```
Comment (kirillplatonov) [inline lib/polaris/view_components/version.rb:3]: Version is updated automatically during gem release. We don't need to change it manually.

Comment (remy727) [reply lib/polaris/view_components/version.rb:3]: Oh, yes. It was my mistake 🤦

Comment (remy727) [reply lib/polaris/view_components/version.rb]: This is reverted

**Comment by remy727** [comment]
I added `demo/app/previews/avatar_component_preview/icon.html.erb` but I don't see new preview in the documentation.

image

image

**Comment by kirillplatonov** [comment]
You need to add a corresponding method to demo/app/previews/avatar_component_preview.rb as well:
https://github.com/baoagency/polaris_view_components/blob/main/demo/app/previews/avatar_component_preview.rb

**Comment by remy727** [comment]
Lookbook is updated.

image

**Review by kirillplatonov — APPROVED** [review]

**Pull request merged**